test/parallel/test-crypto-hash-stream-pipe.js

```diff
@@ -1,20 +1,20 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
```

Comment:
Maybe moving `const assert = require('assert');` after the `if (!common.hasCrypto) {`